share/pvss/pvss.go

```diff
@@ -35,52 +35,65 @@ var errorTooFewShares = errors.New("not enough shares to recover secret")
 var errorDifferentLengths = errors.New("inputs of different lengths")
 var errorEncVerification = errors.New("verification of encrypted share failed")
 var errorDecVerification = errors.New("verification of decrypted share failed")
+var errorChallengeVerification = errors.New("verification of the challenge failed")
+
+type PubRun struct {
```

Comment:
`PubRun` sounds like what I do on a friday night. Would suggest a more descriptive name or a comment explaining it

Reply:
I went for `PubVSSData` and added a comment, hopefully better now